lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -41,4 +50,16 @@ class PokemonOverviewPage extends StatelessWidget {
       ),
     );
   }
+
+  void _noPokemonAvailableDialog(BuildContext context, errorMessage) {

```

Comment:
kindly wrap this snackbar implementation in a separate function and name it with `_showErrorMessageSnackbar` 🙂